src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -284,4 +276,8 @@ public void visionDrive(double left, double right)
         leftTalonLead.set(TalonFXControlMode.Velocity,(left * MAX_VELOCITY));
         rightTalonLead.set(TalonFXControlMode.Velocity,(right * MAX_VELOCITY));
     }
+    public void arcadeDrive(double y, double x)
+    {
+        m_drivetrain.arcadeDrive(y, x);
```

Comment:
Nice! This is what I was thinking to do. Let's rename Y and X to better variables though. By themselves, Y and X don't have much meaning